frontend/src/components/common/comment-item/common/constants.ts

```diff
@@ -1,3 +1,3 @@
-const DEFAULT_COMMENT_COUNT = 0;
+const DEFAULT_COMMENT_VALUE = '';
```

Comment:
should be 0

Reply:
Set hasLikes ? comment.commentReactions.length : ''